SourceControl.md

```diff
@@ -0,0 +1,54 @@
+# General
+
+In the Triple .NET team we use Git for source control management. How you we [Git](https://git-scm.com) - whether it is via a GUI or the CLI - is up to us. Our commits however must stick to the guidelines:
+
+1. We must commit one change per commit
+2. We must adhere to the commit title and -description requirements
+3. We should not commit generated files
+4. We should not commit personal user settings
```

Comment:
'should'? are there valid reasons to sometimes commit user settings?

Reply:
Yes, I'm referring here to CodingGuidelines.md > IDE. There's comments here that go into more detail about this issue - of which I'm not entirely sure yet.